apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/question/repository/WaQuestionRepository.java

```diff
@@ -18,10 +18,11 @@ public List<WaQuestion> findAllByUniqueFields(
             @Param("userDefinedColumnNm") String userDefinedColumnNm,
             @Param("rdbColumnNm") String rdbColumnNm);
 
-    @Query("Select q from WaQuestion q WHERE q.questionNm LIKE %:search% OR q.questionIdentifier LIKE %:search% OR q.questionLabel LIKE %:search% OR q.id =:searchId")
-    public Page<WaQuestion> findAllByNameOrIdentifier(
+    @Query("Select q from WaQuestion q WHERE q.questionNm LIKE %:search% OR q.questionIdentifier LIKE %:search% OR q.questionLabel LIKE %:search% OR q.questionType LIKE %:questionType% OR q.id =:searchId")
+    public Page<WaQuestion> findAllByNameOrIdentifierOrQuestionType(
```

Comment:
If both `search` and `questionType` are provided, the query can return non `questionType` results due to the OR.

I think it could be re-written as 
```
Select q from WaQuestion q WHERE (q.questionNm LIKE %:search% OR q.questionIdentifier LIKE %:search% OR q.questionLabel LIKE %:search% OR q.id =:searchId) AND q.questionType LIKE %:questionType%
```

Reply:
yeah true I'll update

Reply:
@mpeels  updated